site/pages/search/results-en.hbs

```diff
@@ -200,3 +200,8 @@
 		</ul>
 	</div>
 </section>
+
+{{!-- Advanced search link --}}
+<div>
+	<p><a href="https://www.canada.ca/en/sr/srb/sra.html">Advanced search</a></p>
```

Comment:
Replace this: "https://www.canada.ca/en/sr/srb/sra.html" by "#URL-of-the-advance-search-page"